code/game/objects/structures/signs.dm

```diff
@@ -148,4 +148,48 @@
 /obj/structure/sign/directions/evac
 	name = "escape arm"
 	desc = "A direction sign, pointing out which way the escape shuttle dock is."
-	icon_state = "direction_evac"
\ No newline at end of file
+	icon_state = "direction_evac"
+
+/obj/structure/sign/safetyboard //64 by 32 pixels, takes two tiles
+	name = "Station Safety Sign"
+	desc = "Only you can prevent station spaghettification."
+	icon = 'icons/obj/safetyboards.dmi'
+	icon_state = "major_accident"
+	var/value
+	var/record
+	var/record_desc = " The gold trim signifies that we've reached the current record for safety, no pressure!"

```

Comment:
are you implying grammar is not the top priority in any pr i will fight you
:^)